analysis/import/GitLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/gitlogparser/GitLogParser.kt

```diff
@@ -152,4 +154,23 @@ class GitLogParser(
     }
 
     override fun getDialog(): ParserDialogInterface = ParserDialog
+    override fun isUsable(inputFile: String): Boolean {
+        if (inputFile.endsWith(".git")) {
+            return true
+        }
+
+        val searchFile = if (inputFile == "") {
+            File(Paths.get("").toAbsolutePath().toString())
+        } else {
+            File(inputFile)
+        }
+        val result = searchFile.walk()
```

Comment:
I think we can assume to look for the .git folder in the current working directory since this is the default location where git information is located, isn't it? Thus, we could check if there is a folder in the current path named like `.git`
At the moment, all child directories are walked, aren't they?

Reply:
Yes currently we iterate through all child directories and look for a .git file and yes I think you are right, we should probably just look at the given/current directory for a .git file, since we specifically ask for that folder anyway.